addons/account/models/account_bank_statement.py

```diff
@@ -150,7 +150,8 @@ def default_get(self, fields_list):
     )
 
     attachment_ids = fields.Many2many(
-        comodel_name='ir.attachment'
+        comodel_name='ir.attachment',
+        string="Attachments",
```

Comment:
check if translation is already available

Reply:
added the change to account.pot